src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -0,0 +1,161 @@
+package org.broadinstitute.hellbender.utils.fermi;
```

Comment:
Oy.  Are you sure that this is the final package name that everyone's happy with?  It requires changing every file.